src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBUtils.java

```diff
@@ -36,6 +37,50 @@
     private static final String STRAND_BIAS_TABLE_COMBINE = "strand_bias_table";
     private static final String GDB_TYPE_FLOAT = "float";
     private static final String GDB_TYPE_INT = "int";
+    // private static final String MOVE_TO_FORMAT = "move_to_FORMAT";
```

Comment:
I'd rather leave this because I have some work in development that does use it.

Reply:
Actually, I added this based on the email thread and left it commented out to stop the compiler complaining about unused variables. Please feel free to uncomment as necessary.

Reply:
I have uncommitted MOVE_TO_FORMAT too.